browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -1125,8 +1128,37 @@ test.describe('program creation', () => {
     await expect(page.getByRole('checkbox', {name: 'Education'})).toBeChecked()
     await expect(page.getByRole('checkbox', {name: 'Healthcare'})).toBeChecked()
 
-    await test.step('submit program', async () => {
+    await test.step('submit and publish program', async () => {
+      await adminPrograms.submitProgramDetailsEdits()
+      await adminPrograms.publishProgram(programName)
+    })
+
+    await test.step('go to program edit form and check that categories are still pre-selected', async () => {
+      await adminPrograms.gotoViewActiveProgramPageAndStartEditing(programName)
+      await page.getByRole('button', {name: 'Edit program details'}).click()
+      await waitForPageJsLoad(page)
+      await expect(
+        page.getByRole('checkbox', {name: 'Education'}),
```

Comment:
question (non-blocking): can a program have multiple categories?

Reply:
if so, is there an upper limit to it?

Reply:
Yes, a program can have multiple categories.  Right now, there isn't an upper limit, but there will only be 10 category options available (as checkboxes).